### DISCUSSION TITLE
I wanna start learning new stuff related to automation, where do i begin??

### DISCUSSION THREAD

**Discussion by Tanmay-Somani** [community/community#62792] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

see, im new to automation but it always fascinates me so i wanna start learning how to code automation. github pages is such a preference of mine.

**Comment by Mariyajoseph24** [comment, marked as answer]
Hey @Tanmay-Somani, hope you're doing well in your automation learning process, 
Start with "Python" also mentioning a great youtube channel to start learning python as a beginner "techtfq". Python is a great language for automation. You can explore cool RPA tools like UiPath or Automation Anywhere. Take online courses to level up, and don't forget to practice with real projects. Join automation communities for support and learn from others. Read blogs and books for more tips and stay updated with the latest tech trends. Build a portfolio to showcase your skills. Most importantly, have fun on this exciting automation journey! 🚀 Good luck!

**Reply by Tanmay-Somani** [reply]
hey @Mariyajoseph24 I've gone through what you recommended and found them again "explorable", so thank you very much for that. My current read is Automation_with_python. Have you heard of IFTTT? but again thank you for your assisstance.

**Comment by ShonHarsh** [comment]
@Tanmay-Somani 
I want to also recommend the avenue of **UiPath** for automation.  It is a leading platform for automation.  What is particularly helpful is that they provide excellent free learning courses at https://www.uipath.com/rpa/academy.

I recommend starting with the **Automation Explorer** learning path which can be found under Learning Plans.  It is a smaller course and will give you a taste of what the ecosystem is like.

I have many of the projects on my [github](https://github.com/shonharsh) if you wanted to run a project first to see it in action.  For example [this repository](https://github.com/ShonHarsh/RPADev-S07P03-FixMyWorkflow) will automate looking up data on IMDB and documenting it. 

Happy Automation!
